src/main/java/club/sk1er/patcher/mixins/bugfixes/InventoryEffectRendererMixin_FixPotionEffectNumerals.java

```diff
@@ -0,0 +1,44 @@
+package club.sk1er.patcher.mixins.bugfixes;
+
+import club.sk1er.patcher.config.PatcherConfig;
+import club.sk1er.patcher.util.RomanNumerals;
+import net.minecraft.client.renderer.InventoryEffectRenderer;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.potion.PotionEffect;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(InventoryEffectRenderer.class)
+public class InventoryEffectRendererMixin_FixPotionEffectNumerals {
```

Comment:
Instead of 2 Redirects, you can first use [`@ModifyExpressionValue`](https://github.com/LlamaLad7/MixinExtras/wiki/ModifyExpressionValue) to capture the amplifier level then return 0 (instead of 1). Since you return 0, none of the `if`s will be evaluated, and it skips straight to rendering the string, where you can use `@ModifyArg` on the `drawStringWithShadow` call to add the roman numerals.

Minor formatting nitpick, the final closing bracket on the mixin annotations should be one a new line and at the same indentation level as the start of the annotation.

Reply:
I changed with two @ModifyExpressionValue already, is it ok ?

Reply:
That approach is acceptable, however I think the approach I suggest is slightly cleaner (and maybe more compatible with other mods).

Reply:
that doesn't work actually, if I return 0 it will evaluate the two other ifs 
![image](https://user-images.githubusercontent.com/57050655/211127523-cf08cdbc-0ec5-4b8b-8ea8-b2a2c5c22efa.png)

Reply:
Ah right, this is probably fine then.